.github/workflows/ci.yaml

```diff
@@ -175,23 +177,18 @@ jobs:
         repository: input-output-hk/hydra-poc
         token: ${{ secrets.MY_TOKEN || github.token }}
 
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
-      with:
-        folder-path: "hydra-cardano-api, hydra-cluster, hydra-node, hydra-plutus, hydra-prelude, hydra-test-utils, hydra-tui, plutus-cbor, plutus-merkle-tree"
-
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
-      with:
-        folder-path: "."
-        max-depth: 1
```

Comment:
I would keep those one since they're useful, but move them to this new cron job :+1:, links under `docs` are already checked anyway.